src/test/java/com/onaio/steps/activities/HouseholdSummaryActivityTest.java

```diff
@@ -0,0 +1,59 @@
+package com.onaio.steps.activities;
+
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.onaio.steps.R;
+import com.onaio.steps.helper.DatabaseHelper;
+import com.onaio.steps.model.Household;
+import com.onaio.steps.model.InterviewStatus;
+
+import junit.framework.Assert;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@Config(emulateSdk = 16,manifest = "src/main/AndroidManifest.xml")
+@RunWith(RobolectricTestRunner.class)
+public class HouseholdSummaryActivityTest {
+
+    private HouseholdSummaryActivity activity;
+
+    @Before
+    public void setUp() {
+        activity = Robolectric.buildActivity(HouseholdSummaryActivity.class).create().get();
+        insertHouseholds();
+    }
+
+    @Test
+    public void testVerifyAllStatusCount() {
+        activity.onCreate(null);
+
+        LinearLayout container = (LinearLayout) activity.findViewById(R.id.summary_list);
+        Assert.assertEquals(6, container.getChildCount());
+
+        Assert.assertEquals("1", getTotal(container, 0));
+        Assert.assertEquals("1", getTotal(container, 1));
+        Assert.assertEquals("1", getTotal(container, 2));
+        Assert.assertEquals("1", getTotal(container, 3));
+        Assert.assertEquals("1", getTotal(container, 4));
+        Assert.assertEquals("5", getTotal(container, 5));
```

Comment:
Nice test, probably have the returned values different so that if one of the queries broke or switched, the test fails

Reply:
sorry, could you elaborate a bit more than what needs to be changed here?

Reply:
If the code logic for the function `getTotal(container, 0)` broke and returned the value for `getTotal(container, 1)`, the tests wouldn't detect this since the values are the same. This applies for the other assertions in lines 40 to 44. Update so that each query asserts for a different value. This value should not be the same as another value in the same query and it should not be a default (eg. 0) so that they actually test functionality and fail when the code/functionality breaks

Reply:
done.